dockerized/scripts/init.sh

```diff
@@ -1,4 +1,11 @@
 #!/bin/bash
 
 echo $JWT_SECRET > /scripts/.jwt
-executionbackup -p $PORT -n $NODES --listen-addr 0.0.0.0 --jwt-secret /scripts/.jwt --log-level $LOG_LEVEL
\ No newline at end of file
+executionbackup \
+-p $PORT \
+-n $NODES \
+--listen-addr 0.0.0.0 \
+--jwt-secret /scripts/.jwt \
+--log-level $LOG_LEVEL \
+--fcu-majority $FCU_MAJORITY \
+--node-timings
```

Comment:
We shouldn't enable node timings as it's decently verbose and was intended to be used to monitor slow performing nodes, or as a simple debugging solution